packages/components/atoms/f-button/src/components/_tests/Button.test.js

```diff
@@ -5,11 +5,91 @@ describe('Button', () => {
     const actionType = 'button';
 
     it('should be defined', () => {
-        const propsData = { };
+        const propsData = {};
         const wrapper = shallowMount(FButton, { propsData });
         expect(wrapper.exists()).toBe(true);
     });
 
+    describe('methods ::', () => {
+        describe('validateProps ::', () => {
+            it.each([
+                'primary',
+                'secondary',
+                'outline',
+                'ghost',
+                'link'
+            ])('should not throw an error when the component is a standard button (isIcon="false") and buttonType is set to %s', buttonType => {
+                // Arrange
+                const propsData = {
+                    buttonType,
+                    isIcon: false
+                };
+
+                // Act
```

Comment:
Can just do `// Act & Assert` here instead of splitting it.

Reply:
And //Assert right before expect() function

Reply:
Changed 👍